pkg/commands/environment.go

```diff
@@ -30,6 +34,18 @@ func addEnvironmentCmd(topLevel *cobra.Command) {
 	environmentCmd.AddCommand(environmentBumpModuleCmd)
 
 	// flags
+	environmentApplyCmd.Flags().BoolVar(&optFlags.AllNamespaces, "all-namespaces", false, "Apply to all namespaces with -all-namespaces")
+	environmentApplyCmd.Flags().StringVarP(&optFlags.Namespace, "namespace", "n", "", "Namespace which you want to perform the apply")
+	environmentApplyCmd.Flags().StringVar(&optFlags.KubecfgPath, "kubecfg", filepath.Join(homedir.HomeDir(), ".kube", "config"), "path to kubeconfig file")
+	environmentApplyCmd.Flags().StringVar(&optFlags.ClusterCtx, "cluster", "live.cloud-platform.service.justice.gov.uk", "path to kubeconfig file")
+
+	environmentPlanCmd.Flags().IntVar(&optFlags.PRNumber, "pr", 0, "PR number to which ou want to perform the plan")
```

Comment:
Correct spelling 'ou' to 'you'. Really like this concept, but it's hard to understand what PR number is (without context). Perhaps a comment here.

Reply:
Or maybe an example.